internal/configtxgen/encoder/encoder.go

```diff
@@ -181,7 +181,13 @@ func NewChannelGroup(conf *genesisconfig.Profile) (*cb.ConfigGroup, error) {
 // NewOrdererGroup returns the orderer component of the channel configuration.  It defines parameters of the ordering service
 // about how large blocks should be, how frequently they should be emitted, etc. as well as the organizations of the ordering network.
 // It sets the mod_policy of all elements to "Admins".  This group is always present in any channel configuration.
-func NewOrdererGroup(conf *genesisconfig.Orderer) (*cb.ConfigGroup, error) {
+func NewOrdererGroup(conf *genesisconfig.Orderer, channelCapabilities map[string]bool) (*cb.ConfigGroup, error) {
+	if conf.OrdererType == "BFT" {
+		v3Enabled, v3Exists := channelCapabilities["V3_0"]
```

Comment:
`Capabilities` is of type `map[string]bool` so you can just do: 

```
if conf.OrdererType == "BFT" && ! channelCapabilities["V3_0"]` {
        return nil, errors.New("orderer type BFT must be used with V3_0 capability")
}

```

Because the "zero value" of bool is false, so if it's missing or false, `channelCapabilities["V3_0"]` evaluates to false.